content/guides/testing.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Testing Integrations
+kind: guide
+listorder: 6
+autotocdepth: 2
+---
+<style>
+.func-p{margin-left:20px;}
+.func-code{margin-left:40px;}
+</style>
+
+### Overview
+
+Automated unit tests help ensure that your integration is working as designed and guard against regression.
```

Comment:
This should be good right? I think the only difference is guard vs. guards - and it should be guard, since the subject is plural tests. That is "tests guard against regression"